[DM-26816] Add 3 HSC tables from small Qserv

hsinfang opened the pull request.

cbanek commented on docker/tap-schema/1009_hscpdr22020deep_tap_schema.sql:
```diff
@@ -0,0 +1,602 @@
+INSERT INTO tap_schema.schemas11 (schema_name, utype, description, schema_index) VALUES ('hsc_pdr2_2020_08_01', null, 'HSC PDR2 Reprocessing 2020 Spring. DEEP layer', 0);
+INSERT INTO tap_schema.tables11 (schema_name, table_name, table_type, utype, description, table_index) VALUES ('hsc_pdr2_2020_08_01', 'hsc_pdr2_2020_08_01.Object', 'table', null, 'The Object table contains descriptions of the multi-epoch static astronomical objects, in particular their astrophysical properties as derived from analysis of the Sources that are associated with them. Note that fast m ...
```
table -> TABLE here too

cbanek commented on docker/tap-schema/1010_hscpdr22020wide_tap_schema.sql:
```diff
@@ -0,0 +1,602 @@
+INSERT INTO tap_schema.schemas11 (schema_name, utype, description, schema_index) VALUES ('hsc_pdr2_2020_09_00', null, 'HSC PDR2 Reprocessing 2020 Spring. WIDE layer', 0);
+INSERT INTO tap_schema.tables11 (schema_name, table_name, table_type, utype, description, table_index) VALUES ('hsc_pdr2_2020_09_00', 'hsc_pdr2_2020_09_00.Object', 'table', null, 'The Object table contains descriptions of the multi-epoch static astronomical objects, in particular their astrophysical properties as derived from analysis of the Sources that are associated with them. Note that fast m ...
```
table -> TABLE

cbanek reviewed (changes requested):
So this looks good and I think if you change table to TABLE in these, it might work.  If there's a problem with the data, the travis job linked on the PR will fail (it's failing now).  Sometimes the error isn't obvious, but if the TAP_SCHEMA data isnt good then I think things will fail to start properly which thankfully gives an error

cbanek commented:
Also not sure if there's any unquoted single ticks or anything, but those were also other common ingestion problems

cbanek commented:
Oh interesting it looks like it's failing because you're pushing from a fork and therefore don't have my username and password (stored encrypted in this repo).  Let me change the privs so you can push a branch to this repo with your fixes, and you can make a new PR and hopefully that check should work better.  (I still think the table -> TABLE thing will make it fail)

cbanek commented:
Okay not sure if I did this write but I should have "invited you" maybe check your email

hsinfang commented on docker/tap-schema/1008_hscrc2w202030_tap_schema.sql:
```diff
@@ -0,0 +1,601 @@
+INSERT INTO tap_schema.schemas11 (schema_name, utype, description, schema_index) VALUES ('hsc_rc2_w_2020_30_04', null, 'HSC RC2 Reprocessing w_2020_30', 0);
+INSERT INTO tap_schema.tables11 (schema_name, table_name, table_type, utype, description, table_index) VALUES ('hsc_rc2_w_2020_30_04', 'hsc_rc2_w_2020_30_04.Object', 'table', null, 'The Object table contains descriptions of the multi-epoch static astronomical objects, in particular their astrophysical properties as derived from analysis of the Sources that are associated with them. Note that fast ...
```
The latest felis generated 'table" in the lower case.  The older felis seemed to have this in the upper case though.

hsinfang commented:
Thanks for inviting me.  I was able to create a new PR from this repo: https://github.com/lsst-sqre/lsst-tap-service/pull/45   Travis went a lot further though it failed in the end. 

These were mostly generated by felis, except I fixed the database name, which probably was more a problem of the input yaml. 

I also replaced  `schemas` by `tap_schema.schemas11` and so on. 


I just noticed somehow felis double quotes  `"indexed"`; that looks a bit weird to me.

hsinfang commented:
I'm closing this PR and let's continue in 

https://github.com/lsst-sqre/lsst-tap-service/pull/45
